routing/notifications_test.go

```diff
@@ -511,6 +527,13 @@ func TestEdgeUpdateNotification(t *testing.T) {
 				"expected %v, got %v", edgeAnn.TimeLockDelta,
 				edgeUpdate.TimeLockDelta)
 		}
+		if !bytes.Equal(
+			edgeUpdate.ExtraOpaqueData, edgeAnn.ExtraOpaqueData) {
```

Comment:
the linter also expects a line break after this line:
```
		if !bytes.Equal(edgeUpdate.ExtraOpaqueData,
			edgeAnn.ExtraOpaqueData) {

```
Exceptions from the line breaking rule are possible in tests (like for logging https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#exception-for-log-and-error-message-formatting).

For newly added assertions of this type we use `require`, which is shorter and also gives some automatic diffs:
```
		require.Equal(t, edgeAnn.ExtraOpaqueData,
			edgeUpdate.ExtraOpaqueData)
```